plugins/module_utils/deps.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# (c) 2022, Alexei Znamensky <russoz@gmail.com>
+# Copyright (c) 2022, Ansible Project
+# Simplified BSD License (see LICENSES/BSD-2-Clause.txt or https://opensource.org/licenses/BSD-2-Clause)
+# SPDX-License-Identifier: BSD-2-Clause
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+import traceback
+from contextlib import contextmanager
+
+from ansible.module_utils.common.text.converters import to_native
+from ansible.module_utils.basic import missing_required_lib
+
+
+_deps = dict()
```

Comment:
```suggestion
_DEPS = dict()
```

Reply:
Hi Felix, why that? It is not a _constant_, strictly speaking.

Reply:
From my POV it *is* a constant - a constant reference to a mutable object :)

Reply:
If you prefer to keep it as-is, also fine for me :)

Reply:
> From my POV it *is* a constant - a constant reference to a mutable object :)

That would be true for quite a large number of variables. Almost every list or dict out there. ;-)

Reply:
Most variables are not declared at top-level of a Python module :)